### DISCUSSION TITLE
problem with installing v3

### DISCUSSION THREAD

**Discussion by boosting-my-adrenaline** [tailwindlabs/tailwindcss#6775] [General]
I am using React and trying to update to v3 and always get: 

./src/index.css (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-4-1!./node_modules/react-scripts/node_modules/postcss-loader/src??postcss!./src/index.css)
Error: PostCSS plugin tailwindcss requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users

______________________________________________
I had this issue on v2 as well. But there are only solutions for v2.

installing with:
npm install -D tailwindcss postcss autoprefixer
npx tailwindcss init -p

**Comment by adamwathan** [comment]
Make sure you are on create-react-app v5, earlier versions use PostCSS 7 and Tailwind requires PostCSS 8 👍🏻

**Reply by boosting-my-adrenaline** [reply]
a new project runs successfully with create-react-app v5
